src/resources/MachineLearning/ModelListing/ModelListing.ts

```diff
@@ -0,0 +1,14 @@
+import API from '../../../APICore.js';
+import Resource from '../../Resource.js';
+import {MLListingModel} from './ModelListingConfiguration.js';
+
+export default class ModelListing extends Resource {
+    static baseUrl = `/rest/organizations/${API.orgPlaceholder}/machinelearning/configuration/modellisting`;
+
+    list(engineIds?: string[]) {
+        const requestUrl = engineIds?.length
+            ? `${ModelListing.baseUrl}?${engineIds.map((id) => `engineIds=${encodeURI(id)}`).join('&')}`
+            : ModelListing.baseUrl;
+        return this.api.get<MLListingModel[]>(requestUrl);
```

Comment:
Any reason why not using the buildPath utility for the engineIds query parameter?

```suggestion
        return this.api.get<MLListingModel[]>(this.buildPath(ModelListing.baseUrl, {engineIds}));
```

Reply:
You are right! I should use `buildPath` it is so much better.